src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSparkUnitTest.java

```diff
@@ -53,6 +68,16 @@ public void getIntervalsTest() {
     }
 
     @Test(groups = "spark")
+    public void splitIntervalTest() {
+        final List<SVInterval> splitIntervals = Arrays.asList( testBigIntervals )
+                .stream()
+                .flatMap( interval -> FindBreakpointEvidenceSpark.splitInterval( interval, params ).stream() )
+                .collect( java.util.stream.Collectors.toList() );
```

Comment:
Collectors.toList() is fine here:  you know you have a tiny list.  (But import java.util.stream.Collectors so that the code is more compact.)